src/js/views/forms/form/form_views.js

```diff
@@ -284,8 +299,28 @@ const SaveButtonTypeDroplist = Droplist.extend({
   },
 });
 
+const LastUpdatedView = View.extend({
+  className: 'form__last-updated',
+  template: hbs`
+    <div class="form__last-updated-icon">
+      {{far "shield-check"}}
+    </div>
+    <div class="form__last-updated-text">
+      <div class="u-text--overflow">{{ @intl.forms.form.formViews.lastUpdatedView.storedWork }}</div>
+      {{#if updated}}
+        <div class="u-text--overflow">{{formatHTMLMessage (intlGet "forms.form.formViews.lastUpdatedView.updatedAt") updated=(formatDateTime updated "AT_TIME")}}</div>
```

Comment:
This displays a date/time like this:

<img width="227" alt="Screenshot 2023-03-06 at 9 25 35 PM" src="https://user-images.githubusercontent.com/35355575/223312838-b798c191-ce14-4531-ae7f-f0c3d1feb873.png">

I don't think we have any utility functions that do `X seconds ago` like the design doc shows.

Save that for a next iteration?

Reply:
it'd be `{formatDateTime updated "TIME_OR_DAY"}`

Reply:
This doesn't totally match the design docs, but probably good for now:

![Screenshot 2023-03-07 at 9 44 44 AM](https://user-images.githubusercontent.com/35355575/223474533-f2868663-69d9-4ba0-916d-0f23cc479cd3.png)
![Screenshot 2023-03-07 at 9 45 02 AM](https://user-images.githubusercontent.com/35355575/223474546-744d21aa-ace6-4759-bfe5-58c503a0f904.png)

And I also updated the unsaved submission data to `"TIME_OR_DAY"`, as that was updated in the design docs too:

![Screenshot 2023-03-07 at 9 50 27 AM](https://user-images.githubusercontent.com/35355575/223474789-987962ef-1728-4a1f-88cb-02b56a73e73e.png)

Reply:
Oh right.. well where we define `TIME_OR_DAY` we should add:
```js
  AGO_OR_DAY(date) {
    if (date.isSame(dayjs(), 'day')) {
      return date.fromNow();
    }

    return formats.DATE(date);
  },
```

Reply:
Added the `AGO_OR_DAY` in the last push